eprei/ft_transcendence
# Friends can now be managed

Now you can:

- delete friends
- accept friend requests sent by another user
- delete requests that you have sent yourself
- add new friends by choosing them from a list of all users who are not our friends

To test it:

- make clean-database
- make
- login with 42
- http://localhost:8080/api/ (create dummy data)
- go to profile page
- accept, delete, add friends


You should have a pending friend request from another user along with other things
